clang/test/SemaSYCL/invalid_sg_cuda.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple nvptx -internal-isystem %S/Inputs -std=c++2b -verify %s
```

Comment:
Do we need to pass `-std=c++2b` in this test? Asking because it is very common to copy RUN lines from test to test and so we keep unnecessary options to minimum

Reply:
> Do we need to pass `-std=c++2b` in this test? Asking because it is very common to copy RUN lines from test to test and so we keep unnecessary options to minimum

Yeah it is necessary to avoid a warning stating that the attribute requires c++2b when used in that position relative to the host task.